scripts/rssFeedGen.py

```diff
@@ -0,0 +1,95 @@
+from bs4 import BeautifulSoup
+from datetime import datetime
+import time
+from email.Utils import formatdate
+import PyRSS2Gen
+import StringIO
+import re
+
+def extract_releases(html):
+    html_doc = open(html, 'r').read()
+    soup = BeautifulSoup(html_doc, "lxml")
+    divRelease = soup.find("div", {"id":"release-notes"})
+    releases = divRelease.findAll("div", {"id":re.compile("release-")})
+    return releases
+
+def id_to_pubDate(r):
+    id_str = r.findNext("span").get("id")
+    date_str = ''
+    for i in id_str.split():
```

Comment:
this achieves nothing really it puts a string in a list.
What you want to do is split on `-`
So:
```python
date_str = id_str.split('-')[1]
```
remove the loop and everything inside